<launch> 
  <node pkg="planning_environment" type="view_state_validity" respawn="false" output="screen">
    <remap from="collision_map" to="collision_map_occ" />
    <remap from="collision_map_update" to="collision_map_occ_update" />
    <param name="refresh_interval_collision_map" type="double" value="5.0" />
    <param name="refresh_interval_kinematic_state" type="double" value="1.0" />
    <param name="refresh_interval_pose" type="double" value="1.0" />
    <param name="bounding_planes" type="string" value="0 0 1 -0.01" />
  </node>
</launch>
